tests/e2e/utils/switch-date-range.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Changes the currently selected date range.
+ *
+ * Currently only identifiable by the option values.
+ *
+ * @param {string} fromRange The currently selected date range.
+ * @param {string} toRange The new date range to select.
+ */
+export async function switchDateRange( fromRange, toRange ) {
```

Comment:
Would be nice if the field had a unique class name that could be used to target it, instead of `fromRange` 😕 But doesn't seem to be a common thing in the code base.

Reply:
Yes, indeed.